src/shared/install.c

```diff
@@ -894,16 +894,13 @@ static int find_symlinks_in_scope(
                 if (r > 0) {
                         /* We found symlinks in this dir? Yay! Let's see where precisely it is enabled. */
 
-                        r = path_is_config(paths, *p, false);
-                        if (r < 0)
-                                return r;
-                        if (r > 0) {
+                        if (path_equal_ptr(*p, paths->persistent_config)) {
```

Comment:
Perhaps we need to introduce something like `path_is_persistent_config()` or add a new parameter "persistent" to path_is_config() ?

Reply:
I didn't want to do this, because the check is just one line. This way it's very clear what is happening, and if a function is added, things are a bit more obfuscated.